### PR TITLE
English translation updates

### PR DESCRIPTION
**Pull request by milestone-games**
Hi! I've translated some of the strings to English, making a few grammar adjustments as well. Hope you can accept these changes. Thank you!

### CODE REVIEW

File: EUD Editor 3/Language/us-EN.xaml [hunk 1]
```diff
@@ -681,12 +681,12 @@
...
-    <sys:String x:Key="sprites_Sel.Circle Offset_ToolTip">해당 스프라이트를 선택하였을 때 나타나는 선택원의 위아래 위치를 나타냅니다. 크기가 클 수록 선택원이 아래로 내려갑니다.</sys:String>
+    <sys:String x:Key="sprites_Image File_ToolTip">The image referenced by the sprite. Clicking OK will take you to the appropriate image tab.</sys:String>
+    <sys:String x:Key="sprites_Health Bar_ToolTip">This is the size of the health bar that appears when the sprite is selected.</sys:String>
+    <sys:String x:Key="sprites_Unknown2_ToolTip">This data is not used.</sys:String>
+    <sys:String x:Key="sprites_Is Visible_ToolTip">Whether the sprite is visible in-game.</sys:String>
+    <sys:String x:Key="sprites_Sel.Circle Image_ToolTip">Displays the image of the selection circle that appears when the sprite is selected. It depends on the size and shape.</sys:String>
+    <sys:String x:Key="sprites_Sel.Circle Offset_ToolTip">This indicates the top and bottom position of the selection circle that appears when the sprite is selected. The larger the size, the lower the selection circle.</sys:String>
 
 
     <sys:String x:Key="images_GRP File_ToolTip">Display that current image refer to GRP files. Is not work</sys:String>
```
Comment (armoha) [inline EUD Editor 3/Language/us-EN.xaml:692]:
```suggestion
    <sys:String x:Key="images_GRP File_ToolTip">GRP file which is referenced by the image, does not work</sys:String>
```

File: EUD Editor 3/Language/us-EN.xaml [hunk 2]
```diff
@@ -894,64 +893,101 @@
...
 
-    <sys:String x:Key="emptytext">빈 텍스트</sys:String>
+    <sys:String x:Key="emptytext">Blank text</sys:String>
 
     
-    <sys:String x:Key="ChatPluginSetting">ChatEvent 설정</sys:String>
-    <sys:String x:Key="MSQCPluginSetting">MSQC 설정</sys:String>
+    <sys:String x:Key="ChatPluginSetting">ChatEvent Settings</sys:String>
+    <sys:String x:Key="MSQCPluginSetting">MSQC Settings</sys:String>
     <sys:String x:Key="FuncNameUserEdit">UserEdit</sys:String>
```
Comment (armoha) [inline EUD Editor 3/Language/us-EN.xaml:946]:
```suggestion
    <sys:String x:Key="GUIEditorBtn">GUI Editor</sys:String>
```

File: EUD Editor 3/Language/us-EN.xaml [hunk 3]
```diff
@@ -872,16 +872,15 @@
 
 
     <sys:String x:Key="MatchCaseText">Case sendsitive</sys:String>
```
Comment (armoha) [inline EUD Editor 3/Language/us-EN.xaml:874]:
```suggestion
    <sys:String x:Key="MatchCaseText">Case sensitive</sys:String>
```

File: EUD Editor 3/Language/us-EN.xaml [hunk 4]
```diff
@@ -139,7 +139,7 @@
     <sys:String x:Key="PagePaste">Paste page</sys:String>
     <sys:String x:Key="PageReset">Reset page</sys:String>
     <sys:String x:Key="CoverWrite">Paste</sys:String>
-    <sys:String x:Key="Rename">Change name</sys:String>
+    <sys:String x:Key="Rename">Rename</sys:String>
 
     <sys:String x:Key="Save">Save</sys:String>
     <sys:String x:Key="InsertEdd">Run as Edd</sys:String>
```
Comment (armoha) [inline EUD Editor 3/Language/us-EN.xaml:145]:
```suggestion
    <sys:String x:Key="InsertEdd">Run as .edd</sys:String>
```

**Review by armoha — APPROVED** [review]
Looks good to me! Thanks for your contribution on translation. I just left few nitpicks but it's OK to merge as is.

**Comment by milestone-games** [comment]
Thanks for the review! I've accepted your suggestions. Feel free to merge whenever it suits you. 👍

**Comment by armoha** [comment]
I don't have right to merge, only @Buizz can :)

**Pull request merged**